content/library/api/api-reference.md

```diff
@@ -2143,3 +2143,286 @@ else:
 
 </RefCard>
 </TileContainer>
+
+## App testing
+
+<TileContainer>
+
+<RefCard href="/library/api-reference/app-testing/st.testing.v1.apptest" size="full">
+
+#### st.testing.v1.AppTest
```

Comment:
I would love to reduce the number of times we write `st.testing.v1.AppTest` in the docs.

Almost always we want developers to do:

```python
from streamlit.testing.v1 import AppTest

at = Apptest.from_*(...)
```

In most cases, I expect a developer won't even have `import streamlit as st` at the top of a test file

Should we reflect that in any of these code cards and titles? Not required for initial release but something to consider

Reply:
Added to patch